ufersa/plataforma-sabia
# Integrates questions API with front

## Summary


Resolves #531 

![Peek 2020-12-04 16-11](https://user-images.githubusercontent.com/10481028/101206032-47ce0d80-364d-11eb-866f-cdf8a02547c6.gif)
PS: there are only answred questions being listed now.

![Peek 2020-12-04 16-15](https://user-images.githubusercontent.com/10481028/101206048-53b9cf80-364d-11eb-9239-913ead95351a.gif)


## Relevant technical choices



## QA Steps
1. See details of a technology that has no questions
2. Check that there's a message saying so
3. Send a new question
4. See the question listed bellow
***
1. See details of a technology while logged out
2. Check that you can't send a new question
***
1. Go to the questions page at your profile
2. Check that you can see info about the question, refuse or answer it



## Checklist
